src/SFA.DAS.Reservations.Functions.Reservations/ConfirmReservation.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Reservations.Domain.Reservations;
+using SFA.DAS.Reservations.Infrastructure.Attributes;
+using SFA.DAS.Reservations.Infrastructure.NServiceBus;
+
+namespace SFA.DAS.Reservations.Functions.Reservations
+{
+    public class ConfirmReservation
+    {
+        [FunctionName("ConfirmReservation")]
+        public static async Task Run([NServiceBusTrigger(QueueName = "testEndpoint")] ConfirmReservationMessage message, ILogger log, [Inject] IConfirmReservationHandler handler)
```

Comment:
Can we change the queuename to come from a const like the other function - should be in infrastructure proj